--- a/src/backend/base/pyproject.toml
+++ b/src/backend/base/pyproject.toml
@@ -64,7 +64,7 @@ dependencies = [
     "uncurl>=0.0.11,<1.0.0",
     "sentry-sdk[fastapi,loguru]>=2.5.1,<3.0.0",
     "chardet>=7.3.0",
-    "firecrawl-py>=1.0.16,<2.0.0",
+    "firecrawl>=1.0.16,<2.0.0",
     "opentelemetry-api>=1.30.0,<2.0.0",
     "opentelemetry-sdk>=1.30.0,<2.0.0",
     "opentelemetry-exporter-prometheus>=0.50b0,<1.0.0",
